packages/meditrak-server/src/tests/routes/exportSurveyResponses/exportSurveyResponses.test.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import sinon from 'sinon';
+import xlsx from 'xlsx';
+import { Authenticator } from '@tupaia/auth';
+import { buildAndInsertSurveyResponses, buildAndInsertSurveys } from '@tupaia/database';
+import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP } from '../../../permissions';
+import { TestableApp } from '../../TestableApp';
+import { INFO_COLUMN_HEADERS } from '../../../routes/exportSurveyResponses/exportSurveyResponses';
+
+const DEFAULT_POLICY = {
+  DL: ['Public'],
+  KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin', 'Public'],
+  SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+  VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin', 'Donor'],
+  LA: ['Admin'],
+};
+
+const TEST_SURVEY_1_CODE = 'TEST_SURVEY_EXPORT_SURVEY_RESPONSES_1';
+const TEST_SURVEY_1_NAME = 'Export Survey SR 1';
+const TEST_SURVEY_2_CODE = 'TEST_SURVEY_EXPORT_SURVEY_RESPONSES_2';
+const TEST_SURVEY_2_NAME = 'Export Survey SR 2';
+
+const prepareStubAndAuthenticate = async (app, policy = DEFAULT_POLICY) => {
+  sinon.stub(Authenticator.prototype, 'getAccessPolicyForUser').returns(policy);
+  await app.authenticate();
+};
+
+const expectAccessibleExportDataHeaderRow = aoaExportData => {
```

Comment:
`aoa` confused me a bit since I don't know what it stands for... If it does not provide important context to the variable, we can remove it:

```suggestion
const expectAccessibleExportDataHeaderRow = exportData => {
```